python/doc/examples/reliability_sensitivity/sensitivity_analysis/plot_hsic_estimators_ishigami.py

```diff
@@ -0,0 +1,262 @@
+X1 = ot.Uniform(-np.pi, np.pi)
+X2 = ot.Uniform(-np.pi, np.pi)
+X3 = ot.Uniform(-np.pi, np.pi)
+distX = ot.ComposedDistribution([X1, X2, X3])
+
+
+# %%
+# We generate an input sample of size 100 and dimension 3.
+size = 100
+X = distX.getSample(size)
+
+# %%
+# We define the Ishigami model and generate the output sample.
+modelIshigami = ot.SymbolicFunction(
+    ["X1", "X2", "X3"], ["sin(X1) + 5.0 * (sin(X2))^2 + 0.1 * X3^4 * sin(X1)"])
+Y = modelIshigami(X)
+
+
+# %%
+# Setting the covariance models
+# -----------------------------
+#
+# The HSIC algorithms use reproducing Hilbert-Schmidt kernels to estimate independence.
+# For each input variable we choose a covariance kernel. Here we choose a SquaredExponential
+# for all input variables.
+#
+# They are all stored in a CovarianceModelCollection of size d+1 where d is the number of
+# input variables. The remaining one is for the output variable.
+covarianceList = ot.CovarianceModelCollection()
+
+# %%
+for i in range(3):
+    Xi = X.getMarginal(i)
+    inputCovariance = ot.SquaredExponential(1)
+    inputCovariance.setScale(Xi.computeStandardDeviation())
+    covarianceList.add(inputCovariance)
+
+# %%
+# Likewise we define a covariance model associated to the output variable.
+outputCovariance = ot.SquaredExponential(1)
+outputCovariance.setScale(Y.computeStandardDeviation())
+covarianceList.add(outputCovariance)
+
+
+# %%
+# The Conditional HSIC estimator
+# ------------------------------
+#
+# In this section we use a preprocessing on the input variable: that is the conditional
+# analysis. To do so, one has to work with a weight function.
+#
+
+# %%
+# Defining a weight function
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# We define a weight function on the input variables for the conditional
+# analysis. In this example we use the function :math:`\exp{(-d/s)}` where d is the distance
+# to a well-choosen interval.
+
+# %%
+# We first define a critical domain: in our case that is the :math:`[5,+\infty[` interval.
+criticalDomain = ot.Interval(5, np.inf)
+
+# %%
+# We have access to the distance to this  domain thanks to the
+# :class:`~openturns.DistanceToDomainFunction` class.
+dist2criticalDomain = ot.DistanceToDomainFunction(criticalDomain)
+
+# %%
+# We define the parameters in our function from the output sample
+s = 0.1 * Y.computeStandardDeviation()[0]
+
+# %%
+# We now define our weight function by composition of the parametrized function and
+# the distance function.
+f = ot.SymbolicFunction(["x", "s"], ["exp(-x/s)"])
+phi = ot.ParametricFunction(f, [1], [s])
+weightFunction = ot.ComposedFunction(phi, dist2criticalDomain)
+
+
+# %%
+# Choosing an estimator
+# ^^^^^^^^^^^^^^^^^^^^^
+#
+# After having define the covariance models and the weight function one has to select an
+# appropriate estimator for the computations.
+#
+# Two estimators are proposed:
+# - an unbiased estimator through the HSICUStat class
+# - a biased estimator through the HSICVStat class
```

Comment:
```suggestion
#
# - an unbiased estimator through the :class:`~openturns.HSICUStat` class
# - a biased estimator through the :class:`~openturns.HSICVStat` class
```
Without the empty line, the "-" are not understood as bullet points, which results in a formatting error.

Reply:
I would suggest to specify for the V-stat:
"a biased, but asymptotically unbiased, estimator through the HSICVStat". This is an important property which makes the use of V-stat estimators interesting, since they can be easier to estimate than U-stat but still show consistent results asymptotically.